contributing.md

```diff
@@ -65,7 +65,7 @@ Dockerfile is added so users on all systems can get involved in development. Win
 To build the image:
 
 ```
-$ docker build -t pydp:test .
+$ docker build -t pydp:test ...
```

Comment:
Any specific reason for using three dots?

Reply:
Ooops forgot to take it out. I did that to trigger a change in the copied files. will submit an edit soon.